src/Dapr.Workflow/WorkflowLoggingService.cs

```diff
@@ -0,0 +1,88 @@
+// ------------------------------------------------------------------------
+// Copyright 2022 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+namespace Dapr.Workflow
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// Defines runtime options for workflows.
+    /// </summary>
+    internal sealed class WorkflowLoggingService : IHostedService
+    {
+        private readonly ILogger<WorkflowLoggingService> logger;
+        private static HashSet<string>? registeredWorkflows;
+        private static HashSet<string>? registeredActivities;
+        private LogLevel logLevel = LogLevel.Debug;
+
+        public WorkflowLoggingService(ILogger<WorkflowLoggingService> logger)
+        {
+            var value = Environment.GetEnvironmentVariable("DAPR_LOG_LEVEL");
+            logLevel = string.IsNullOrEmpty(value) ? LogLevel.Debug : (LogLevel)Enum.Parse(typeof(LogLevel), value);
```

Comment:
Rather than making the logging level of its entires configurable, I'd suggest using a level that makes the most sense for the information being displayed (information?), and then rely on MS logging's mechanisms for configuring which logs are written, which would seem more in line with ASP.NET logging.